training/model_management/components/model_sanity_validation/spec.yaml

```diff
@@ -0,0 +1,49 @@
+$schema: https://azuremlschemas.azureedge.net/latest/commandComponent.schema.json
+
+name: mlflow_model_local_validator
+version: 0.0.1
+type: command
+
+is_deterministic: True
+
+display_name: MLFlow model local validator
+description: Validates if a MLFLow model is locally loadable and inferencable
```

Comment:
Consider changing "inferencable" to something more like "usable for inferencing" or related.

Reply:
Done